a11y.js

```diff
@@ -9,21 +9,20 @@ const services = require('./cypress/support/config/services');
 const pageWidths = [360];
 const baseUrl = 'http://localhost:7080';
 
-const getPageTypes = (service) =>
-  pathOr(null, [service, 'pageTypes'], services);
+const getPageTypes = service => pathOr(null, [service, 'pageTypes'], services);
```

Comment:
```suggestion
const getPageTypes = service => path([service, 'pageTypes'], services);
```

Reply:
and line 1: `const path = require('ramda/src/path');` (out of range for github suggestion 😢 )